apps/events.md

```diff
@@ -811,3 +813,49 @@ order: {
   totalTax: "3.2"
 }
 ```
+
+### shippingMethod:select
+
+To get more information about this event, you can use the following snippet:
+
+```js
+if (window.eComEventTarget) {
+  window.eComEventTarget.addEventListener('shippingMethod:select', function (event) {
+    console.log('shippingMethod:select', {
+      shippingMethod: event.detail.shippingMethod,
+    })
+  })
+}
+```
+
+Here's an example of what you can get:
```

Comment:
@dkluever22 I prefer "Here is" instead of the "Here's"-option. I also have a question in general. In your version, I would expect a list of possible things I could get. However, I'm not sure if this applies here as the following text sounds more like a general example. So maybe like this instead?

```suggestion
Here is an example of what you may get:
```

Reply:
@AliceStatler I used the wording that had been used for other events in the event section. So if we change the wording in this case I would like to adapt it to all the other events as well, to make it consistent.
I get your point, though. @bolsu The examples provided, are those example responses or just response extracts? Could we write something like "Here is an example response you can/may/could get:" to make it clearer or would that be wrong from a technical perspective? Also, which verb would you preferably use?

Reply:
@bolsu Thanks a lot, Tim 😀 Your suggestion works for me. What do you think, @AliceStatler?

Reply:
@dkluever22 @bolsu I also think "Here is an example response" works well here. :)